.github/workflows/gatk-tests.yml

```diff
@@ -32,15 +32,15 @@ jobs:
         env:
           GCP_CREDENTIALS: ${{ secrets.GCP_CREDENTIALS }}
         if: "${{ env.GCP_CREDENTIALS != '' }}"
-        run: echo "::set-output name=defined::true"
+        run: echo defined=true >> $GITHUB_OUTPUT
```

Comment:
These still work for outputs? I remember I had to use those eldrich invocations as messy as they were to pass variables across jobs.

Reply:
Bleh the report upload link looks broken now? 
 https://storage.googleapis.com/hellbender-test-logs/build_reports/8241/merge_4420157996.1/tests/testOnPackagedReleaseJar/index.html

Reply:
Well on the plus side it looks like it was already broken to follow the index.html link posted by a run job...